mandatory/3-playing-computer.js

```diff
@@ -28,7 +28,7 @@ const f2 = function (a, b) {
 
 console.log(x);
 console.log(a);
-console.log(b);
+
```

Comment:
well done Hengame , but i think for this exercise there is some questions that you forgot to answer :)